### PR TITLE
removed useless html

### PR DESCRIPTION
**Pull request by lyudmyla-horbyk**
#8

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -474,8 +474,6 @@ template {
 .header__language-selector {
   margin-left: 10px;
   margin-right: 5px;
-}
-.language-selector {
```
Comment (probil) [inline css/style.css:478]: Why you decided to remove this class `language-selector`?

File: css/style.css [hunk 2]
```diff
@@ -1158,10 +1157,25 @@ template {
   align-items: flex-end;
   justify-content: space-around;
   box-sizing: border-box;
-  padding-bottom: 4px;
+  padding-bottom: 9px;
   padding-left: 5px;
   padding-right: 4px;
 }
+.footer__google-icon {
+  background-image: url("../img/google.png");
```
Comment (probil) [inline css/style.css]: [optional] 
since icons are really small I would suggest adding them inline as `base64`

File: index.html [hunk 3]
```diff
@@ -140,10 +140,9 @@
               <img src="./img/img-slider.jpg" alt="Слайдер Товарів" />
             </div>
             <div class="slider__bottom">
-              <span class="slider__bottom__text"
-                >Каталог постільних <br />комплектів</span
-              >
-
+              <div class="slider__bottom__text">
+                <span>Каталог постільних комплектів</span>
```
Comment (probil) [inline index.html]: why `span` is needed here? 🤔 
I would put everything in `<div>`

File: index.html [hunk 4]
```diff
@@ -598,60 +595,53 @@
...
+              src="./img/footer-logo.png"
+              alt="Знак якості"
+            />
           </div>
           <p class="footer__quality-mark__text">
             © 2013 Baby Brok. Всі права захищені
-            <br />
-            <br />
+          </p>
+          <p class="footer__quality-mark__text">
```
Comment (probil) [inline index.html:636]: nice! 👍

File: index.html [hunk 5]
```diff
@@ -53,7 +53,7 @@
                   placeholder="Пошук по магазину"
                 />
               </div>
-              <div class="header__language-selector language-selector">
+              <div class="header__language-selector">
```
Comment (probil) [inline index.html]: Why you decided to remove class `language-selector`?  It's part of the BEM. 
On line 57 you use `language-selector__button` which is an element but there is no block for it

File: index.html [hunk 6]
```diff
@@ -598,60 +595,53 @@
...
             © 2013 Baby Brok. Всі права захищені
-            <br />
-            <br />
+          </p>
+          <p class="footer__quality-mark__text">
             Всі торгові марки є торговими марками їх відповідних власників
           </p>
         </div>
         <div class="footer__more-information">
           <div class="footer__social-networks">
```
Comment (probil) [inline index.html]: I believe having `<nav>` here is a bit better then `<div>`. What do you think?

File: index.html [hunk 7]
```diff
@@ -574,14 +573,12 @@
                 diam nonumy eirmod tempor invidunt ut labore et dolore magna
                 aliquyam erat, sed diam voluptua.
               </a>
-              <br />
```
Comment (probil) [inline index.html:577]: 👍

File: css/style.css [hunk 8]
```diff
@@ -1158,10 +1157,25 @@ template {
...
+  height: 27px;
+  width: 28px;
+}
+.footer__vkontakte-icon {
+  background-image: url("..//img/vkontakte.png");
```
Comment (probil) [inline css/style.css]: double slash. But check my comment above ⬆️

**Review by probil — CHANGES_REQUESTED** [review]

File: css/style.css (hunk 1 above)
Comment (lyudmyla-horbyk) [reply css/style.css:478]: Because this class hasn't styles.

File: css/style.css (hunk 2 above)
Comment (lyudmyla-horbyk) [reply css/style.css]: Okay, I agree.

File: index.html (hunk 3 above)
Comment (lyudmyla-horbyk) [reply index.html]: Agree, no sense use `span`

File: index.html (hunk 5 above)
Comment (lyudmyla-horbyk) [reply index.html]: Sorry, it's my fault.

File: index.html (hunk 6 above)
Comment (lyudmyla-horbyk) [reply index.html]: I think it's good idea, because we have some `a` tegs. Thanks!

File: css/style.css (hunk 8 above)
Comment (lyudmyla-horbyk) [reply css/style.css]: Okay!

**Comment by lyudmyla-horbyk** [comment]
@probil Please, check the changes!

File: index.html (hunk 6 above)
Comment (probil) [reply index.html]: ** tags

File: css/style.css (hunk 1 above)
Comment (probil) [reply css/style.css:478]: Makes sense 🤔

File: index.html (hunk 5 above)
Comment (probil) [reply index.html]: It's ok. Do treat it so seriously :)

**Review by probil — APPROVED** [review]

File: index.html (hunk 5 above)
Comment (lyudmyla-horbyk) [reply index.html]: Okay)

**Pull request merged**